funderfinder/sources/numfocus.py

```diff
@@ -12,42 +15,60 @@
 """
 
 
-def get_funding_stats(search_params: dict) -> dict:
-    """
-    Determines whether a project is sponsored or affiliated with NumFOCUS based on our scraped dataset and
-    a project name, slug, or github owner and repo name
-    :param search_params: Dict of user-provided metadata that we can use to match a numfocus project
-    :return: Dict of funding stats
-    """
-    is_affiliated = False
-    with open(
-        os.path.join(
-            os.path.dirname(os.path.abspath(__file__)), "..", "data", "numfocus.jsonl"
+class NumFocusFinder(Finder):
+    name = "NumFOCUS"
+
+    @staticmethod
+    def get_funding_stats(search_params: dict) -> Union[dict, None]:
+        """
+        Determines whether a project is sponsored or affiliated with NumFOCUS based on our scraped dataset and
+        a project name, slug, or github owner and repo name
+        :param search_params: Dict of user-provided metadata that we can use to match a numfocus project
+        :return: Dict of funding stats
+        """
+        is_affiliated = False
+        with open(
+            os.path.join(
```

Comment:
This is the sort of formatting where I think `black` occasionally goes too far.

What about:

```python
fpath = os.path.dirname(os.path.abspath(__file__)), ..", "data", "numfocus.jsonl")
with open(fpath) as f:
    ...
```